deploy/firesim

```diff
@@ -271,6 +272,8 @@ def main(args):
     # we elastically spin instances up/down. we can easily get re-used IPs with
     # different keys. also, probably won't get MITM'd
     env.disable_known_hosts = True
+    # do not ever ask for input interactively
+    env.abort_on_prompts = True
```

Comment:
@sagark do you know of a usecase where we want fabric to query the user for "stuff"?  There are many things that can cause it to ask for different things, including failure to provide hostnames,etc.  Since they use `raw_input()` to print their prompts and all stdout is going only to the log, I thought it made the most sense to tell fabric to bail out if it needs to ask for anything interactively.

Reply:
I remember discussing this in a dev meeting and Sagar saying that this was fine.